### PR TITLE
[15.0][ADD] website_conditional_visibility_user_group: New module

### PR DESCRIPTION
**Pull request by chienandalu**
Please review @pedrobaeza @pilarvargas-tecnativa 

cc @Tecnativa TT50008

### CODE REVIEW

**Review by pilarvargas-tecnativa — APPROVED** [review]
Functional review, thanks

File: website_conditional_visibility_user_group/readme/DESCRIPTION.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Thsi module adds extra granularity in the visibility of the website snippets allowing
```
Comment (pedrobaeza) [inline website_conditional_visibility_user_group/readme/DESCRIPTION.md]:
```suggestion
This module adds extra granularity in the visibility of the website snippets allowing
```

**Review by pedrobaeza — APPROVED** [review]
Code review and tested in runboat

**Pull request merged**